deploy/README.md

```diff
@@ -0,0 +1,313 @@
+# Wikibase Suite
+
+Wikibase Suite (WBS) is a containerised, production ready [Wikibase](https://wikiba.se) system, that allows you to self host a knowledge graph similar to Wikidata.
+
+> 🔧 This document is for people wanting to self host the full Wikibase Suite using the Wikibase Suite Deployment Kit. If you are looking for individual WBS Service Images, head over to [hub.docker.com/u/wikibase](https://hub.docker.com/u/wikibase).
+
+> 💡 This document assumes basic Linux administration and docker/docker-compose knowledge.
+
+## What is in the box?
+
+This WBS Deployment Kit uses the following WBS Service Images, all of them packaged as Docker Images:
+
+- **[Wikibase](https://hub.docker.com/r/wikibase/wikibase)** MediaWiki packaged with the Wikibase extension and other commonly used extensions.
+- **Job Runner** The MediaWiki [JobRunner](https://www.mediawiki.org/wiki/Manual:Job_queue#Cron) service which uses the same Wikibase container as above.
+- **[MariaDB](https://hub.docker.com/_/mariadb)** Database service for MediaWiki and Wikibase.
+- **[Elasticsearch](https://hub.docker.com/r/wikibase/elasticsearch)** Search service used by MediaWiki.
+- **[WDQS](https://hub.docker.com/r/wikibase/wdqs)** Wikidata Query Service allowing SPARQL queries.
+- **[WDQS Frontend](https://hub.docker.com/r/wikibase/wdqs-frontend)** Web Frontend to run SPARQL queries.
+- **[wdqs-proxy](https://hub.docker.com/r/wikibase/wdqs-proxy)** A middle layer for WDQS which serves to filter requests and make the service more secure.
+- **[WDQS Updater](https://www.mediawiki.org/wiki/Wikidata_Query_Service/User_Manual#runUpdate.sh)** Keeps the WDQS data in sync with Wikibase.
+- **[Quickstatements](https://hub.docker.com/r/wikibase/quickstatements)** A web based tool to import and manipulate large amounts of data.
+- **[Traefik](https://hub.docker.com/_/traefik)** A reverse proxy handling TLS termination and ACME certificate renewal.
+
+## Quickstart
+
+### Requirements
+
+> 💡 If you want to run a quick test on a machine that has no public IP address (e.g. your local machine), check our [FAQ item](#can-i-host-wikibase-suite-locally) below.
+
+#### Hardware
+
+- Network connection with a public IP address
+- AMD64 architecture
+- 8 GB RAM
+- 4 GB free disk space
+
+#### Software
+
+- Docker 22.0, or greater
+- Docker Compose 2.10, or greater
+- git
+
+#### Domain Names
+
+You need three DNS records that resolve to your machines IP address.
```

Comment:
machine's - possessive

Reply:
Thanks a lot. Fixed in the original PR. If you want to take part in the docs review, please head over to #683 

This PR was intended for the demo only. So closing it now.